src/main/java/com/jfrog/ide/idea/navigation/NavigationService.java

```diff
@@ -49,8 +49,10 @@ public void addNavigation(DependencyNode treeNode, PsiElement navigationTargetEl
         }
         NavigationTarget navigationTarget = new NavigationTarget(navigationTargetElement, document.getLineNumber(navigationTargetElement.getTextOffset()), componentName);
         Set<NavigationTarget> navigationTargets = navigationMap.get(treeNode);
+
```

Comment:
all the changes are spaces? we can remove then...

Reply:
um i dont see spaces in my code..